components/proxy/src/main/java/com/hotels/styx/StyxPipelineFactory.java

```diff
@@ -87,7 +90,7 @@ private RoutingObject configuredPipeline(RoutingObjectFactory routingObjectFacto
                 })
                 .orElseGet(() -> {
                     Registry<BackendService> backendServicesRegistry = (Registry<BackendService>) services.get("backendServiceRegistry");
-                    return new StaticPipelineFactory(environment, backendServicesRegistry, plugins, requestTracking);
+                    return new StaticPipelineFactory(environment, backendServicesRegistry, plugins, eventLoopGroupFactory, requestTracking);
```

Comment:
It seems we are only passing along the configured eventLoopGroupFactory for 'static pipelines'  (I haven't really checked the other code path, maybe I'm missing something)

Reply:
Static pipeline is more important because it implements the documented styx functionality. 

All other uses are for routing objects that are more or less experimental at the moment, and we'd need to think about how to go forward with them in near future. I see that as a separate increment. I suspect once the routing object model is finalised in styx 1.1 we'll end up deprecating the `ProxyToBackend` and `BackendServiceProxy` objects as unnecessary.